assignment.md

```diff
@@ -11,63 +11,123 @@ If the version control concepts or `git` commands are still confusing to you, co
+    git remote show origin
+
+    #output
+    * remote origin
+      Fetch URL: https://github.com/WCM-datascibasics/version_control_assignment.git
+      Push  URL: https://github.com/WCM-datascibasics/version_control_assignment.git
+      HEAD branch: main
+      Remote branches:
+        main                tracked
+        revert-3-submission tracked
+        testing             tracked
+      Local branch configured for 'git pull':
+        main merges with remote main
+      Local ref configured for 'git push':
+        main pushes to main (up to date)
+
     ```
 
 
- 2. Add a second remote destination named `upstream` to your assignment repository, `upstream` is a common name used for the repository that people have forked from, in our case, the `upstream` will be at `https://github.com/WCM-datascibasics/version_control_assignment`. List the new list of remotes after you've added the new remote. 
- 
+ 2. Add a second remote destination named `upstream` to your assignment repository, `upstream` is a common name used for the repository that people have forked from, in our case, the `upstream` will be at `https://github.com/WCM-datascibasics/version_control_assignment`. List the new list of remotes after you've added the new remote.
+
     ```bash
-    # list the new list of remotes here
+    # Command
+    git remote add upstream https://github.com/WCM-datascibasics/version_control_assignment.git
+
+    # outputs
+    git remote -v
+    origin	https://github.com/WCM-datascibasics/version_control_assignment.git (fetch)
+origin	https://github.com/WCM-datascibasics/version_control_assignment.git (push)
+upstream	https://github.com/WCM-datascibasics/version_control_assignment.git (fetch)
+upstream	https://github.com/WCM-datascibasics/version_control_assignment.git (push)
     ```
 
  3. Save your modifications to this file so far and create a commit indicating you've answered the first 2 questions. Then try pushing the changes to the `upstream` destination on GitHub. What happens? Explain in your own words why does this happen? What are the benefits of having this `upstream` remote added when working collaboratively (Hint: read git command outputs!)?
 
-    > Your answer here
+    > I was denied access, as I am not listed as a collaborator on upstream. Having upstream you are not committing to master, so bugs do not effect working code. The admin can gate what goes into master more easily.
 
 
  4. Now clone the repository for the [class website](https://github.com/WCM-datascibasics/wcm-datascibasics.github.io), and in the class website repository:
     - a. Explore the version history by visualizing it as a graph (remember the dog meme?).
         ```bash
-        # git command and output here
+        # git cloning command
+        git clone https://github.com/WCM-datascibasics/wcm-datascibasics.github.io
+
+        #I dont have visual studio and do not have the space on my machine to add it, so I cannot install git graph.
```

Comment:
if you already have `git` working... `git log --graph --decorate` is simply a command... just like `git status` or whatever... this should've worked...? You don't need visual studio to use `git` commands.... not sure what happened. -10%.